diff --git a/modules/core.js b/modules/core.js
index 4c56da1358ff1b66e3c21643309b418ef08c725d..0fa2f4255c85d7a5c8d412bbc6fae37bce3db22f 100644
--- a/modules/core.js
+++ b/modules/core.js
@@ -129,7 +129,7 @@
 	};
 
 	// Virtual file system for browser
-	tau_file_system = {
+	var tau_file_system = {
 		// Current files
 		files: new TauDirectory("/", "/", null),
 		// Open file
@@ -169,7 +169,7 @@
 	};
 
 	// User input for browser
-	tau_user_input = {
+	var tau_user_input = {
 		buffer: "",
 		get: function( length, _ ) {
 			var text;
@@ -191,7 +191,7 @@
 	};
 
 	// User output for browser
-	tau_user_output = {
+	var tau_user_output = {
 		put: function( text, _ ) {
 			console.log( text );
 			return true;
@@ -202,7 +202,7 @@
 	};
 
 	// User error for browser
-	tau_user_error = {
+	var tau_user_error = {
 		put: function( text, _ ) {
 			(console.error || console.log)( text );
 			return true;
@@ -213,7 +213,7 @@
 	};
 
 	// Virtual file system for Node.js
-	nodejs_file_system = {
+	var nodejs_file_system = {
 		// Open file
 		open: function( path, type, mode ) {
 			var fd, fs = require('fs');
@@ -281,7 +281,7 @@
 	};
 
 	// User input for Node.js
-	nodejs_user_input = {
+	var nodejs_user_input = {
 		buffer: "",
 		get: function( length, _ ) {
 			var text;
@@ -298,7 +298,7 @@
 	};
 
 	// User output for Node.js
-	nodejs_user_output = {
+	var nodejs_user_output = {
 		put: function( text, _ ) {
 			process.stdout.write( text );
 			return true;
@@ -309,7 +309,7 @@
 	};
 
 	// User error for Node.js
-	nodejs_user_error = {
+	var nodejs_user_error = {
 		put: function( text, _ ) {
 			process.stderr.write( text );
 			return true;
